.github/workflows/build.yml

```diff
@@ -0,0 +1,27 @@
+name: 'Test'
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - '*'
```

Comment:
I think you can save two lines by leaving this out as is done in https://github.com/getsentry/symbolicator/pull/235/files. This way is pretty readable though, so could be preferable.